pymllm/quantization/kernels/int8_activation_triton.py

```
@@ -0,0 +1,82 @@
+
+    Returns:
+        x_q: INT8 quantized tensor, same shape as x.
+        scales: Per-token scales, shape = x.shape[:-1] + (1,).
+    """
+    assert x.is_contiguous(), "Input must be contiguous"
+
+    M = x.numel() // x.shape[-1]
+    N = x.shape[-1]
+    x_q = torch.empty_like(x, device=x.device, dtype=torch.int8)
+    scales = torch.empty(
+        x.shape[:-1] + (1,), device=x.device, dtype=scale_dtype
```

Ruff (0.15.12)

[warning] Consider `(*x.shape[:-1], 1)` instead of concatenation

Replace with `(*x.shape[:-1], 1)`

(RUF005)